scripts/process_notebooks.py

```diff
@@ -136,12 +136,16 @@ def should_process(path):
         # Ensure that Colab metadata dict exists and enforce some settings
         add_colab_metadata(nb, nb_name)
 
-        # Clean the original notebook and save it to disk
+        # Clean the original notebook and save it to disk if it belongs to tutorials
```

Comment:
```suggestion
         # Write the original notebook back to disk, clearing outputs only for tutorials
```

There was a slight grammatical ambiguity in the original comment